### PR TITLE
feat: Add missing field to tests project in the github pipelines

### PR DESCRIPTION
**Pull request by hosekpeter**
Jira: https://keboola.atlassian.net/browse/PSGO-375

**Changes:**
- updated go-utils version
- added field 'backend' to test project in the github pipelines

-----------

### CODE REVIEW

**Comment by hosekpeter** [comment]
@michaljurecko I need to find out, which projects are for `bigquery`

File: .github/workflows/test-e2e-cli.yml [hunk 1]
```diff
@@ -13,33 +13,33 @@ env:
...
-      {"host":"connection.north-europe.azure.keboola.com","project":10890,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10890_TOKEN }}"},
-      {"host":"connection.north-europe.azure.keboola.com","project":10891,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10891_TOKEN }}"},
-      {"host":"connection.north-europe.azure.keboola.com","project":10892,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10892_TOKEN }}"}
+      {"host":"connection.keboola.com","backend":"snowflake","project":8764,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8764_TOKEN }}"},
+      {"host":"connection.keboola.com","backend":"snowflake","project":8765,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8765_TOKEN }}"},
+      {"host":"connection.keboola.com","backend":"snowflake","project":8766,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8766_TOKEN }}"},
+      {"host":"connection.north-europe.azure.keboola.com","backend":"snowflake","project":10890,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10890_TOKEN }}"},
```
Comment (michaljurecko) [inline .github/workflows/test-e2e-cli.yml:19]: Please, add `backend` field after the `stagingStorage` field, to improve readability.

**Review by michaljurecko — COMMENTED** [review]
There is no BigQuery project in this repo.

(I didn't remember that either, but I checked now.)

The next task will therefore add at least 2 BigQuery projects to each section.
Can you please create the task in Jira?

Stack: `connection.europe-west3.gcp.keboola.com`, project type: `BigQuery Production`

File: .env.dist [hunk 2]
```diff
@@ -1 +1 @@
-TEST_KBC_PROJECTS=[{"host":"connection.keboola.com","project":1234,"stagingStorage":"s3","token":"<token>"},...]
+TEST_KBC_PROJECTS='[{"host":"connection.keboola.com","backend":"snowflake/bigquery", "project":1234,"stagingStorage":"s3","token":"<token>"},...]'
```
Comment (michaljurecko) [inline .env.dist:1]: Can you please use same fields sort in the README? Thanks!

**Comment by hosekpeter** [comment]
> There is no BigQuery project in this repo.
> 
> (I didn't remember that either, but I checked now.)
> 
> The next task will therefore add at least 2 BigQuery projects to each section. Can you please create the task in Jira?
> 
> Stack: `connection.europe-west3.gcp.keboola.com`, project type: `BigQuery Production`

done

**Review by michaljurecko — APPROVED** [review]
LGTM

**Pull request merged**